tests/Xamarin.Android.Bcl-Tests/TestRunner.Core/Extensions.Bool.cs

```diff
@@ -0,0 +1,10 @@
+namespace Xamarin.Android.UnitTests
+{
+	public static partial class Extensions
+	{
+		public static string YesNo (this bool b)
```

Comment:
Why do we have this method? `YesNo()` isn't used anywhere.

Reply:
It is used [here](https://github.com/xamarin/xamarin-android/pull/1075/files#diff-1ba463e47d47e7922c319ac47da53a89R483)